File: Fisica/src/Fisica.Website/ClientApp/src/app/shared/models/instituicao.model.ts
import { Endereco } from "./endereco.model";

export class Instituicao {
  constructor(
    public id?: number,
    public nome?: string,
    public descricao?: string,
    public email?: string,
    public site?: string,
    public endereco?: Endereco
  ) { }
}

File: Fisica/src/Fisica/Models/InstituicaoModel.cs
namespace Fisica.Models
{
    public class InstituicaoModel
    {
        public long? Id { get; set; }
        public string? Nome { get; set; }
        public string? Descricao { get; set; }
        public string? Email { get; set; }
        public string? Site { get; set; }
        public long? EnderecoId { get; set; }
    }
}
